pytorch_lightning/utilities/model_helpers.py

```diff
@@ -69,3 +74,19 @@ def is_overridden(
     parent_code = parent_attr.__code__
 
     return instance_code != parent_code
+
+
+def get_model_size(model: nn.Module) -> int:
```

Comment:
let's include the units in the name so readers of the code immediately know that the returned value is in MB
```suggestion
def get_model_size_mb(model: nn.Module) -> int:
```